frontend/packages/ceph-storage-plugin/src/components/ocs-install/install-page.tsx

```diff
@@ -90,33 +97,39 @@ const InstallCluster: React.FC<InstallClusterProps> = ({ match }) => {
     return sanitizedMode;
   };
 
-  const getStep = (offset: number = 0) => {
-    const searchParams = new URLSearchParams(window.location.search.slice(1));
-    const step = parseInt(searchParams.get('step'), 10) || 1;
-    const sanitizedStep = step && step <= 5 - offset && step >= 1 ? step : 1;
-    return sanitizedStep;
-  };
-
   const getParamString = (step: number, mode: number) => {
     const searchParams = new URLSearchParams(window.location.search.slice(1));
     searchParams.set('step', step.toString());
     searchParams.set('mode', mode.toString());
     return searchParams.toString();
   };
 
+  const getAnchor = (step: number, mode: number) => `~new?${getParamString(step, mode)}`;
+
+  const getStep = (offset: number = 0) => {
+    const searchParams = new URLSearchParams(window.location.search.slice(1));
+    const step = parseInt(searchParams.get('step'), 10) || 1;
+    const sanitizedStep = step && step <= 5 - offset && step >= 1 ? step : 1;
+    return sanitizedStep;
+  };
+
   const getIndex = (searchSpace: any, search: string, offset: number = 0) => {
     const index = Object.values(searchSpace).findIndex((el) => el === search);
     return index - offset + 1;
   };
 
-  const getAnchor = (step: number, mode: number) => `~new?${getParamString(step, mode)}`;
-
   const handleModeChange = (event: React.FormEvent<HTMLInputElement>) => {
     const { value } = event.currentTarget;
     const modeIndex = getIndex(MODES, value);
-    history.push(
-      `~new?${getParamString(getIndex(CreateStepsSC, CreateStepsSC.DISCOVER), modeIndex)}`,
-    );
+    // `modeIndex` can have one of the following numerical values:
+    // 1: INTERNAL
+    // 2: ATTACHED_DEVICES
+    // 3: EXTERNAL
```

Comment:
I expect the comment should indicate that we are setting this for attached devices installation where the starting step has to be different with the presence of a no-prov storage class,

Reply:
Noted. Making changes.